app/src/main/res/values/strings.xml

```diff
@@ -672,4 +672,10 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="about_missing_stars">Currently, the star count is not shared across devices, but answers are added directly to the OpenStreetMap database.</string>
     <string name="how_to_get_stars">Choose one of the markers on the map and answer the question. This way you will improve OpenStreetMap data and get stars. If you see no markers you may need to zoom out the map.</string>
     <string name="quest_cyclewayPartSurface_title">What\'s the surface of the cycleway here?</string>
+    <string name="quest_generic_answer_differs_along_the_way">"Differs along the way…"</string>
+    <string name="quest_split_way_description">If it differs along the way, the first step is to split up the way. After that, the quest can be answered for each split part separately.\nNote that the quests for the split parts may appear mach later.\nSplit it now?</string>
+    <string name="quest_split_way_tutorial">Tap on the way to split it at the point(s) where the answer differs. Try to be as precise as possible, you can zoom in as usual.</string>
+    <string name="quest_split_way_too_imprecise">"Please zoom in further"</string>
```

Comment:
This appears also at the maximum possible zoom.

Reply:
Hm. The message appears if you tapped multiple nodes at once with your finger. If the nodes are absurdedly close to each other, then it wouldn't be possible to split up the way.

Reply:
I'd love to just set the maximum zoom further up. But The default maximum zoom in tangram-es is 20.5 and it cannot be turned up further (and only starting at the new tangram-es version that we do not use).

At zoom 20.5, the diameter of the tap of the finger is about 3 meters in our latitude and about 5 on the equator. So, that is about still 4 zoom levels away from achieving the OSM 7-decimals-precision.

So the best choice here is to instead of requiring to further zoom in at already that high of a zoom level, just choose the node that is closest to the center of the tap position.

Will work on it. Thank you!

Reply:
Implemented it. Would you mind testing it again at the position where you discovered it?

Reply:
@matkoniecz